content/en/developers/metrics/metrics_type.md

```diff
@@ -36,73 +30,73 @@ But the following metric submission types are accepted:
 * [SET](?tab=set#metric-type-definition)
 * [TIMERS](?tab=timers#metric-type-definition)
 
+****RUTH: I'm struggling with what to call these types. Intuitively it's difficult for customers to understand why looking up a metric submitted as a distribution in the metrics summary page would return the metric type of a Gauge in the Metrics Summary detail sidepanel****
+Once a metric is submitted, there are 3 metric storage types within the Datadog web application:
 
-Choosing a submission metric type depends on your use case and how your application is designed:
-
-* If you want to count states, you would use a `GAUGE` type. For instance, if you want to track the number of healthy nodes accross your infrastructure.
-* If you want to count events, you would use a `COUNT` or a `RATE` type. For instance, if you want to track the number of requests hitting your webservers.
-* If you want to combine multiple sources of submission, you would use a `HISTOGRAM` or a `DISTRIBUTION` type. For instance, if you want to track the 99 percentile response time of different hosts for a given service
+* `COUNT`
+* `RATE`
+* `GAUGE`
 
 Refer to the [Metric Type Definition](#metric-type-definition) section below to learn more about the different metrics types available. Also, refer to the [Submission Types and Datadog In-App Types Section](#submission-types-and-datadog-in-app-types) to see how each metric type behaves between its submission and its storage within Datadog.
 
 ## Metric type definition
 
-To better understand the different metrics types, what they represent, and how to manipulate them within Datadog, consider the following example:
-
-You have two web servers: `server:web_1` and `server:web_2`. Both deal with HTTP requests over time.
-
-They both receive:
-
-  * 10 requests for the first 30 seconds, then
-  * 20 requests for the next 30 seconds, then
-  * 0 request for the last 30 seconds.
+To better understand the different metrics types, what they represent, and how to modify them within Datadog, consider the following example:
 
-Then the pattern starts again.
+You have two web servers: `server:web_1` and `server:web_2`. Both web servers continuously receive:
+  * 10 HTTP requests for the first 30 seconds, then
+  * 20 HTTP requests for the next 30 seconds, then
+  * 0 HTTP requests for the next 30 seconds.
```

Comment:
For this example, I think we need to not use 0. It’s not clear if that means no value was received, or if a 0 was submitted. 

If that truly was a 0 value, no interpolation is needed, there’s a value - 0. 

If it’s a NaN, we’ll .fill(last) for all of these. I think it starts to muddy the waters a bit.

Reply:
The other bit is, I think using a 0 here, if we want to keep these examples, is confusing. If it means "no value submitted", we'll have a NaN, interpolation kicks in, that's one case. Gauge/Count/Rate will all three have fill(last, 5 minutes) applied, and rolllup avg.

If the 0 is an actual 0 submitted by the Agent, all of those types will actually preserve the 0 in-app, no interpolation.